values.yaml

```diff
@@ -274,6 +274,12 @@ syncCatalog:
   # prepended with "consul-". (Consul -> Kubernetes sync)
   k8sPrefix: null
 
+  # consulServicePrefix is the service prefix which preprends itself 
+  # to Kubernetes services registered within Consul
+  # For example, "k8s-" will register all services peprended with "k8s-". 
+  # (Kubernetes -> Consul sync)
+  consulServicePrefix: null
```

Comment:
Does it make sense to name this `consulPrefix` to be consistent with `k8sPrefix`?